habitat-baselines/habitat_baselines/config/socialnav/social_nav.yaml

```diff
@@ -0,0 +1,259 @@
+# @package _global_
+
+defaults:
+  - /benchmark/multi_agent: hssd_spot_human_social_nav
+  - /habitat_baselines: habitat_baselines_rl_config_base
+
+  - /habitat/simulator/sim_sensors@habitat_baselines.eval.extra_sim_sensors.third_rgb_sensor: third_rgb_sensor
+  - /habitat_baselines/rl/policy/obs_transforms@habitat_baselines.rl.policy.agent_0.obs_transforms.add_virtual_keys:
+    - add_virtual_keys_base
+  - /habitat_baselines/rl/policy/obs_transforms@habitat_baselines.rl.policy.agent_1.obs_transforms.add_virtual_keys:
+    - add_virtual_keys_base
+  - /habitat_baselines/rl/policy@habitat_baselines.rl.policy.agent_0: monolithic
+  - /habitat_baselines/rl/policy@habitat_baselines.rl.policy.agent_1: hl_fixed_social_nav
+  - /habitat_baselines/rl/policy/hierarchical_policy/defined_skills@habitat_baselines.rl.policy.agent_1.hierarchical_policy.defined_skills: oracle_skills_human_multi_agent
+  - /habitat/task/actions@habitat.task.actions.agent_1_oracle_nav_randcoord_action: oracle_nav_action
+
+  - /habitat/task/measurements:
+    - social_nav_reward
+    - object_to_goal_distance
+    - articulated_agent_force
+    - force_terminate
+    - articulated_agent_colls
+    - rot_dist_to_goal
+    - dist_to_goal
+    - nav_to_pos_succ
+    - social_nav_seek_success
+    - social_nav_stats
+
+  - /habitat/task/lab_sensors:
+    - relative_resting_pos_sensor
+    - target_start_sensor
+    - goal_sensor
+    - joint_sensor
+    - is_holding_sensor
+    - end_effector_sensor
+    - target_start_gps_compass_sensor
+    - target_goal_gps_compass_sensor
+    - localization_sensor
+    - humanoid_joint_sensor
+    - humanoid_detector_sensor
+
+  - /habitat/task/lab_sensors:
+    - has_finished_oracle_nav
+
+  - /habitat/task/lab_sensors@habitat.task.lab_sensors.agent_0_goal_to_agent_gps_compass: nav_goal_sensor
+  - override /habitat/simulator/sensor_setups@habitat.simulator.agents.agent_0: rgbd_head_rgbdp_arm_agent
+  - override /habitat/task/actions@habitat.task.actions.agent_0_base_velocity: base_velocity_non_cylinder
+  - _self_
+
+hydra:
+  job:
+    name: 'social_nav'
+
+habitat:
```

Comment:
All task level overrides (under `habitat`) belong in the task config, not the training config.